api/src/main/java/io/strimzi/api/kafka/Crds.java

```diff
@@ -75,6 +76,14 @@ private static CustomResourceDefinition crd(Class<? extends CustomResource> cls)
             listKind = KafkaConnectS2IAssembly.RESOURCE_LIST_KIND;
             group = KafkaConnectS2IAssembly.RESOURCE_GROUP;
             version = KafkaConnectS2IAssembly.VERSION;
+        } else if (cls.equals(KafkaUser.class)) {
+            kind = KafkaUser.RESOURCE_KIND;
+            crdApiVersion = KafkaUser.CRD_API_VERSION;
+            plural = KafkaUser.RESOURCE_PLURAL;
+            singular = KafkaUser.RESOURCE_SINGULAR;
+            listKind = KafkaUser.RESOURCE_LIST_KIND;
+            group = KafkaUser.RESOURCE_GROUP;
+            version = KafkaUser.VERSION;
```

Comment:
you will need to also set the shortname here. My KafkaTopic PR changes this slightly though, so it's probably worth waiting for me to merge that before you go refactoring this too much.

Reply:
Yeah ... well ... when are you merging? :-o

Reply:
Ok, this should be now rebased after your PR.